### PR TITLE
Requirements update

### PR DESCRIPTION
**Pull request by davyskiba**
Updated app requirements with things we learned on the way regarding backend integration, using secrets, and GPS accuracy.
Moved milestone 3 to "future milestones" section

### CODE REVIEW

**Review by arturjaworski — APPROVED** [review]

**Review by JakubJankowski — APPROVED** [review]

File: app-requirements.md [hunk 1]
```diff
@@ -44,23 +44,28 @@ For this milestone we improve on the simplistic solution by bring in additional
...
 Subscriber app (the consumer / customer):
 
-- Must use the backend service to generate an order for delivery - TBC whether this is simply by entering order id into the app, or perhaps by getting a list of active orders that can be accepted
+- Must use the backend service to generate an order for delivery by entering order starting location and the destination
 - Should configure the resolution based on the user’s behaviour (if the app is in the background, or the user is looking at a different page: we don’t need a high resolution)
-- Should ideally, when the tracker is 10 m from the address, remove the map and mention the food is here
-- Android:
-  - Must source Google Maps API key from the backend service
+- Should ideally, when the tracker is 50 m from the address, remove the map and mention the food is here. 10 m threshold proved to be too small given the gps accuracy.
+- Must display last location update interval as well as a rolling average of the last five update intervals
```
Comment (QuintinWillison) [inline app-requirements.md]: I would perhaps have this as _should_ rather than _must_.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
Could you please revert the change where you've moved the `Future Milestones` H2 up. The document no longer makes sense with that change, especially given `Milestone 3: Enhanced` is still an H2 also.

I think the existing structure was still fine. This is a roadmap, not a promise to deliver the work, so a third milestone (even if we don't deliver it anytime soon) is fine to stay where it was.

File: app-requirements.md (hunk 1 above)
Comment (davyskiba) [reply app-requirements.md]: updated d75467141b4d11c4ad8f528003f7b9bce59fcfe0

**Comment by davyskiba** [comment]
@QuintinWillison moved the header back d75467141b4d11c4ad8f528003f7b9bce59fcfe0

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**